libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Connector.Teams;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Teams
+{
+    public class TeamsInfo
+    {
+        private IConnectorClient _connectorClient;
+        private ITeamsConnectorClient _teamsConnectorClient;
+
+        public TeamsInfo(ConnectorClient connectorClient)
+        {
+            if (connectorClient != null)
+            {
+                _connectorClient = connectorClient;
+                _teamsConnectorClient = new TeamsConnectorClient(connectorClient.BaseUri, connectorClient.Credentials, connectorClient.HttpClient);
+            }
```

Comment:
Is there an else for this?

Reply:
no this can be null - if it is then the methods throw later